lib/tags/profile-settings.tag

```diff
@@ -325,6 +327,7 @@
                     let successMessage = i18n.usernameSuccessMessage;
                     self.refs.username.classList.remove('graphjs-error');
                     self.refs.username.classList.add('graphjs-success');
+                    self.refs.username.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.username refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above